<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input Bilibili Xml String</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="inputEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="parseButton">
     <property name="text">
      <string>Parse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parse danmaku</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="outputEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
